sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultAccessControlClientBuilder.java

```diff
@@ -186,6 +194,20 @@ public KeyVaultAccessControlAsyncClient buildAsyncClient() {
         return new KeyVaultAccessControlAsyncClient(vaultUrl, buildPipeline, serviceVersion);
     }
 
+    private HttpPipelinePolicy getAndValidateRetryPolicy() {
+        if (retryPolicy != null && retryOptions != null) {
+            throw logger.logExceptionAsWarning(
+                new IllegalStateException("'retryPolicy' and 'retryOptions' cannot both be set"));
+        }
+        if (retryPolicy != null) {
+            return retryPolicy;
+        } else if (retryOptions != null) {
+            return new RetryPolicy(retryOptions);
+        } else {
+            return new RetryPolicy();
+        }
+    }
```

Comment:
Since this is required in every builder and this method only uses core types to validate, can this be moved to a utility method in core?